src/m365/spfx/commands/project/project-azuredevops-pipeline-model.ts

```diff
@@ -0,0 +1,38 @@
+export interface azureDevOpsPipeline {
+  name?: string,
```

Comment:
Let's end each line in the declaration with a `;` rather than a `,`

Reply:
done, 
along the way I corrected the same mistake in github workflow interfaces. We had similar problem there as well